package.json

```diff
@@ -62,19 +63,23 @@
     "eslint-plugin-import": "^2.9.0",
     "eslint-plugin-react": "^7.13.0",
     "eslint-plugin-react-hooks": "^1.6.0",
-    "file-loader": "^4.2.0",
-    "html-webpack-plugin": "^3.2.0",
     "prettier": "^1.15.3",
     "prop-types": "^15.7.2",
-    "raw-loader": "^1.0.0",
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
     "rimraf": "^3.0.0",
+    "rollup": "^1.21.4",
+    "rollup-plugin-babel": "^4.3.3",
+    "rollup-plugin-buble": "^0.19.8",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-image": "^1.0.2",
+    "rollup-plugin-livereload": "^1.0.4",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-replace": "^2.2.0",
+    "rollup-plugin-serve": "^1.0.1",
+    "rollup-plugin-terser": "^5.1.2",
     "tslint": "^5.12.1",
-    "typescript": "^3.2.2",
-    "webpack": "^4.40.2",
```

Comment:
It looks like we haven't removed the `ROOT/webpack*.js` files yet in this PR?

Reply:
I was going to do that once all build format PRs are in but I'm happy to do it now